sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ImplementISerializableCorrectly.cs

```diff
@@ -231,7 +231,7 @@ private static bool IsCallingBaseConstructor(IMethodSymbol constructorSymbol)
             typeSymbol.IsPubliclyAccessible() &&
             typeSymbol.AllInterfaces.Any(IsOrImplementsISerializable);
 
-        private static bool HasSerializableAttribute(ITypeSymbol typeSymbol) =>
+        private static bool HasSerializableAttribute(ISymbol typeSymbol) =>
```

Comment:
While reverting the changes in this file, this change was still kept, although it is unrelated to the PR

Reply:
Indeed this seems more like a random change but I think it would be just a loss of time to make this back `ITypeSymbol`, trigger the azure job and wait for it to finish 😄